docs/home/howto/LOGGING.md

```diff
@@ -1,119 +1,124 @@
+in [official documentation](https://opensearch.org/docs/latest).
 
-The main and most important decisions to take before you deploy cluster are:
+The main and most important decisions to take before you deploy the cluster are:
 
-1) How many Nodes are needed
-2) How big machines and/or storage data disks need to be used
+ - how many nodes are needed
+ - how big machines and/or storage data disks need to be used
 
-These parameters are defined in yaml file, and it's important to create a big enough cluster.
+These parameters can be defined in manifest yaml file. It is important to create a big enough cluster.
 
 ```yaml
 specification:
+  [..]
   components:
     logging:
-      count: 1    #  Choose number of nodes
+      count: 1    #  Choose number of nodes that suits your needs
+      machines:
+      - logging-machine-n
+  [..]
 ---
 kind: infrastructure/virtual-machine
 title: "Virtual Machine Infra"
-name: logging-machine
+name: logging-machine-n
 specification:
-  size: Standard_DS2_v2    #  Choose machine size
+  size: Standard_DS2_v2    #  Choose a VM size that suits your needs
 ```
 
-If it's required to have Elasticsearch which works in cluster formation configuration, except setting up more than one
+If it's required to have OpenSearch instance which works in cluster formation configuration, except setting up more than one
 machine in yaml config file please acquaint dedicated
-support [article](https://opendistro.github.io/for-elasticsearch-docs/docs/elasticsearch/cluster/) and adjust
-Elasticsearch configuration file.
+support [article](https://opensearch.org/docs/latest/troubleshoot/index/) and adjust
+OpenSearch configuration file.
 
-At this moment Opendistro for Elasticsearch does not support plugin similar
-to [ILM](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-lifecycle-management.html), log rotation
-is possible only by configuration created in Index State Management.
+We also want to strongly encourage you to get familiar with a bunch of plugins and policies available along with OpenSearch with the following ones among them:
 
-`ISM - Index State Management` - is a plugin that provides users and administrative panel to monitor the indices and
+`ISM - Index State Management` - is a plugin that allows users and administrative panel to monitor the indices and
 apply policies at different index stages. ISM lets users automate periodic, administrative operations by triggering them
 based on index age, size, or number of documents. Using the ISM plugin, can define policies that automatically handle
-index rollovers or deletions. ISM is installed with Opendistro by default - user does not have to enable this. Official
+index rollovers or deletions. Official plugin
 documentation is available
-in [Opendistro for Elasticsearch website](https://opendistro.github.io/for-elasticsearch-docs/docs/im/ism/).
+[here](https://opensearch.org/docs/latest/im-plugin/ism/index/).
```

Comment:
Unnecessary line split

Reply:
Sure, I found a lot of things like this in this doc.